DZ8_LOOPS/index.js

```diff
@@ -0,0 +1,158 @@
+//1. Вивести на сторінку в один рядок через кому числа від 10 до 20.
+const numsArr = []
```

Comment:
1. `console.log` need to call after the loop

```
for () {
  // do math
}

console.log(
            `1. Вивести на сторінку в один рядок через кому числа від 10 до 20.\n${numsArr.join(', ')}`)
```

2. To isolate each subtask you can use IIFE function or block
```
// Block
{
 // here the subtask
}

// IIFE
(() => {
 // here the subtask
})()
```